src/main/java/uk/gov/hmcts/reform/hmc/config/MessageSenderConfiguration.java

```diff
@@ -36,6 +40,13 @@ public void sendMessage(String message, MessageType messageType, String caseId)
             log.debug("Message has been sent to the Queue {}", applicationParams.getQueueName());
         } catch (Exception e) {
             log.error("Error while sending the message to queue:{}", e.getMessage());
+            log.error(
+                "Error occurred during service bus processing. Service:{} . Type: {}. Method: {}. Hearing ID: {}.",
+                HMC_HMI_INBOUND_ADAPTER,
+                TYPE_INBOUND,
```

Comment:
should this be inbound when it is writing to a queue?

Reply:
to become outbound